swap_meet/vendor.py

```diff
@@ -1,2 +1,219 @@
-class Vendor:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+# Wave 1- SJ
+# 1. created the Vendor class
+# 2. one attribute named inventory, empty list by default
+
+class Vendor: 
+    
+    def __init__(self, inventory=None):
+        self.inventory =  inventory if inventory is not None else []
+    
+    
+    # Wave 1- SJ
+    # 3. Created instance method named add, which takes in one item
+    # 3a. This method adds an item to the inventory
+    # 3b. This method returns the item that was added
+
+    def add(self, added_item):
+        self.added_item = added_item
```

Comment:
So, I see you did this a couple times in your project (`self.added_item`), this is actually going to create an attribute called `added_item` on the object that `add` was called on, you want to be careful of adding attributes to things unintentionally.  You could change the code you have to the following and get the same functionality we expected:
```python
def add(self, add_item):
     self.inventory.append(added_item)
     return added_item
```

Reply:
oooooooh okay, didn't know i could do that. This is cool!